lua/wire/zvm/zvm_opcodes.lua

```diff
@@ -1576,9 +1576,9 @@ end
 ZVM.OpcodeTable[267] = function(self)   --MLOOKAT
   local seg1code = self.EmitOperandSegment[1] and "0" or "VM.DS"
   local seg2code = self.EmitOperandSegment[2] and "0" or "VM.DS"
-  self:Dyn_Emit("$L EYE    = VM:ReadVector3f($2 + %s+0",seg2code)
-  self:Dyn_Emit("$L CENTER = VM:ReadVector3f($2 + %s+3",seg2code)
-  self:Dyn_Emit("$L UP     = VM:ReadVector3f($2 + %s+6",seg2code)
+  self:Dyn_Emit("$L EYE = VM:ReadVector3f($2 + VM."..(self.EmitOperandSegment[2] or "DS").."+0)")
```

Comment:
These can still use %s right?

Reply:
Actually yeah, that was how I fixed it initially(just adding the right parenthesis), I think I copy & pasted the original lines to test if the behavior was the same and forgot to put the fixed ones that use seg2code back